### PR TITLE
Copy ETH addresses to clipboard 

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

Adds icon to ETH address tooltips to copy address to clipboard. 

Design channel thread: https://discord.com/channels/775859454780244028/930063712444887071

## Screenshots or screen recordings

https://user-images.githubusercontent.com/96150256/149033340-310ec0e9-1ed0-49f0-ad95-da36b3fd4b09.mp4

## Acceptance checklist

- [x] I have evaluated the [Approval Guidelines](../../CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](../../CONTRIBUTING.md#browser-support).

### CODE REVIEW

File: src/components/shared/FormattedAddress.tsx [hunk 1]
```diff
@@ -97,8 +98,11 @@ export default function FormattedAddress({
       trigger={['hover', 'click']}
       title={
         <span>
-          <span style={{ userSelect: 'all' }}>{address}</span>{' '}
-          <EtherscanLink value={address} type="address" />
+          <span style={{ userSelect: 'all' }}>
+            <EtherscanLink value={address} type="address" />
+          </span>{' '}
+          {/* <EtherscanLink value={address} type="address" />{' '} */}
```
Comment (peripheralist) [inline src/components/shared/FormattedAddress.tsx:104]: can we remove this?

File: src/components/shared/EtherscanLink.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { LinkOutlined } from '@ant-design/icons'
+// import { LinkOutlined } from '@ant-design/icons'
```
Comment (peripheralist) [inline src/components/shared/EtherscanLink.tsx:1]: and this

**Review by peripheralist — APPROVED** [review]
looks great! just some code i think we can delete

File: src/components/shared/CopyTextButton.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { CopyOutlined } from '@ant-design/icons'
+import { Tooltip } from 'antd'
+import { useState } from 'react'
+
+// Copies a given text to clipboard when clicked
```
Comment (peripheralist) [inline src/components/shared/CopyTextButton.tsx:5]: nice! we should be better about adding descriptive comments to components

**Comment by johnnyd-eth** [comment]
@peripheralist thanks done!

**Pull request merged**